doc/HOWTO_forks.md

```diff
@@ -8,7 +8,14 @@ set up remote repositories on each side.
 
 _While in your fork's directory_
 
-`git remote add upstream git@github.com:tablexi/nucore-open.git` (You should only need to do this once.)
+You should only need to do these once:
+
+```
+git remote add upstream git@github.com:tablexi/nucore-open.git
+chmod +x ./bin/merge_describer
```

Comment:
I _think_ (and I could be wrong), if you have the execute permission added when you commit the file, you don't need to add it again when someone else clones it.